src/components/PageCount.tsx

```diff
@@ -0,0 +1,15 @@
+import Box from "@mui/material/Box";
```

Comment:
Should this PageCount component be part of this commit? Is it part of the pagination work?